gitpod-io/website
# Add Hacktoberfest blog post

## Description
Add a blog post about hacking with Gitpod for [Hacktoberfest 2022](https://hacktoberfest.com/).

With 🧡 from IDE

## How to test


See https://deploy-preview-2780--gitpod-kumquat.netlify.app/blog/hacktoberfest-2022

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2780"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2780"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
